melodies_monet/plots/BACKUP/surfplots_backup.py

```diff
@@ -0,0 +1,1078 @@
+# Copyright (C) 2022 National Center for Atmospheric Research and National Oceanic and Atmospheric Administration
```

Comment:
Do we need to keep track of these backup files on GitHub? I would prefer these just be in your own code for documentation, but we do not add it to the main GitHub repo.

Reply:
no. ler me delete it

Reply:
I just delete it.